test/resolvers_test.go

```diff
@@ -190,7 +190,7 @@ spec:
 	if err := WaitForPipelineRunState(ctx, c, prName, timeout,
 		Chain(
 			FailedWithReason(pod.ReasonCouldntGetTask, prName),
-			FailedWithMessage("requested resource 'https://api.hub.tekton.dev/v1/resource/Tekton/task/git-clone-this-does-not-exist/0.7/yaml' not found on hub", prName),
+			FailedWithMessage("requested resource 'https://artifacthub.io/api/v1/packages/tekton-task/tekton-catalog-tasks/git-clone-this-does-not-exist/0.7.0' not found on hub", prName),
```

Comment:
instead of changing this test, can we leave it as is to use the tekton hub, and add `type: tekton` to the test case?

then we can add another test case that defaults to artifact hub? 

want to make sure that both scenarios are covered in the end-to-end tests

Reply:
ah I see, how soon are we planning to remove it? so users can use hub resolver with their own tekton hub instances?

Reply:
Yeah, exactly! Based on the current progress (https://github.com/tektoncd/hub/issues/667), I think we can deprecate `https://api.hub.tekton.dev` in late Dec 2022 or early 2023. And yes, user must configure their own instance of Tekton Hub to use `tekton type` 😄